### DISCUSSION TITLE
Expired certificate on https://dtcenter.ucar.edu

### DISCUSSION THREAD

**Discussion by venitahagerty** [dtcenter/METplus#1643] [Installation] [answered]
I use curl to retrieve the data I use for automated testing. The certificate has expired which is giving me an error.

hagerty-lt:temp venita.hagerty$ curl 'https://dtcenter.ucar.edu/dfiles/code/METplus/METdatadb/sample_data-met_out_v9.1.tgz' -o m91.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

**Comment by JohnHalleyGotway** [comment, marked as answer]
@venitahagerty thanks for the heads up. We see the same behavior when trying to access the other METplus data files at https://dtcenter.ucar.edu/dfiles/code. @georgemccabe has contacted the NCAR/RAL IT dept to ask them to update the certificate for that host.

**Reply by JohnHalleyGotway** [reply]
@venitahagerty, the RAL IT dept plans to update the certs for that machine tomorrow (6/7/2022) when the person who usually does it is available. Please let me know if it's still not working by COB tomorrow. Thanks again, and sorry for the hassle that this one slipped through the cracks.

**Reply by JohnHalleyGotway** [reply]
OK, the certificate issue is now fixed, and the setup has been updated to auto-renew. Thanks again for letting us know. I'll mark this one as being answered.
